docs/cheatsheet.rst

```diff
@@ -6,12 +6,13 @@ Cheatsheet
 
 Order of Precedence of Operators
 ================================
+
 .. include:: types/operator-precedence-table.rst
 
-.. index:: assert, block, coinbase, difficulty, prevrandao, number, block;number, timestamp, block;timestamp, msg, data, gas, sender, value, gas price, origin, revert, require, keccak256, ripemd160, sha256, ecrecover, addmod, mulmod, cryptography, this, super, selfdestruct, balance, codehash, send
+.. include:: abi; decode, abi; encode, abi; encodePacked, abi; encodeWithSelector, abi; encodeCall, abi; encodeWithSignature
```

Comment:
`include` is for including files. This was supposed to be `index`. Please note that the `b_docs` job is failing because of this. The PR must pass all the CI checks before it can be merged.